functional_programming.ipynb

```diff
@@ -806,10 +792,10 @@
     "This is a function that takes a function `f(x, y)` of two arguments and an iterable `it` and applies the function to every element in the iterable *cumulatively* to produce one value.\n",
     "It works in the following way:\n",
     "\n",
-    "- The first argument of `f`, `x` is the current value of the accumulation. At first, this corresponds to the first element of `it`\n",
-    "- The second argument of `f`. `y` is the current element of the iterable. At first, this corresponds to the second element of `it`.\n",
+    "- The first argument of `f`, `x`, is the **current value of the accumulation**. At the beginning, this corresponds to the first element of `it`\n",
+    "- The second argument of `f`, `y`, is the **current element of the iterable**. At the beginning, this corresponds to the second element of `it`.\n",
     "\n",
-    "Because of this behavior, this function is useful to compute sums or similar aggregations over a list. In Python, this function is available in the `functools` [module](https://docs.python.org/3/library/functools.html) as a part of the Python standard library.\n",
+    "Because of this behavior, this function is useful to compute sums or similar aggregations over a list. In Python, this function is available in the [`functools` module](https://docs.python.org/3/library/functools.html), which is part of Python's standard library.\n",
```

Comment:
```suggestion
    "Because of this behaviour, this function is handy to compute sums or similar aggregations over a list. In Python, this function is available in the [`functools` module](https://docs.python.org/3/library/functools.html), part of Python's standard library.\n",
```